frontend/packages/console-plugin-sdk/src/codegen/plugin-integration-tests.ts

```diff
@@ -0,0 +1,38 @@
+import { resolve } from 'path';
+import * as _ from 'lodash';
+import {
+  PluginPackage,
+  filterActivePluginPackages,
+  resolvePluginPackages,
+} from './plugin-resolver';
+
+const consoleIntegrationTestPrefix = '@console/internal-integration-tests';
+
+const extractFullPath = (testPath: string, pluginPath: string) =>
+  testPath.startsWith(consoleIntegrationTestPrefix)
```

Comment:
I'd prefer a more defensive approach - check if the path is relative, if so do the `resolve` thing, if not - check if the path starts with `@console` prefix, if so do the `substring` thing, if not - fail.